webpack/scenes/RedHatRepositories/index.scss

```diff
@@ -0,0 +1,9 @@
+$pf-black-200: #ededed;
```

Comment:
https://www.patternfly.org/styles/color-palette/

Reply:
I was hoping there was a way to use this - https://github.com/patternfly/patternfly/blob/e31f1c7da704c00fa9320ad5de8aa37b584f13f6/src/sass/converted/patternfly/_color-variables.scss#L3.  Something along the lines of

```sass
@import "patternfly/_color-variables.scss";

background-container-gray {
  background-color: $pf-black-200;
}
```
Do you mind seeing if this is possible to avoid the duplication of the color specification.